src/document/types.ml

```diff
@@ -89,11 +89,22 @@ and Block : sig
     | Math of Math.t
     | Verbatim of string
     | Raw_markup of Raw_markup.t
+    | Table of t Table.t
 
   and list_type = Ordered | Unordered
 end =
   Block
 
+and Table : sig
+  type alignment = [ `Left | `Center | `Right ]
+
+  type 'a t = {
+    data : ('a * [ `Header | `Data ]) list list;
+    align : alignment option list option;
```

Comment:
Meh.
This leads to ugly code in the html backend, and some "normalisation" in the latex one.
Perhaps we could normalize when we create the elements, instead of when we render them; so `align : alignment option list`?

Furthermore: I assume the polymorphic variants are here because the parser emits them, but:
1. I'm not a fan
2. If we're already doing some datatype mapping from the parser to odoc types, let's go a bit further:  
   ```ocaml
   type alignment  = Default | Left | Center | Right
   type 'a t = {
     date : ...;
     align: alignment list;
   }
   ```

Reply:
I agree with removing the polymorphic variant.
I don't mind using `Default` instead of `None`.
I think I delayed the "compute the number of columns when alignment is unspecified" since it's not always needed, but I agree it's not ideal, so I implemented your proposition, which seems better.